api/src/Controller/BaseController.php

```diff
@@ -502,13 +502,17 @@ public function getDepartment($id)
                 $select->subselect()
                     ->columns('GROUP_CONCAT(Email)')
                     ->from('EMails')
-                    ->whereEquals(['DepartmentID' => 'd1.DepartmentID'])
+                    ->where('DepartmentID = d1.DepartmentID')
```

Comment:
Same here.

Reply:
When calling `api/department` the value of `email` was null with the original code and now correctly reflects the `email` list with the new changes.

Unless `email` was supposed to be null here, I believe there may have been an error in the syntax.

Reply:
The statement is indeed wrong.
should be:
`->whereEquals(['DepartmentID' => d1.DepartmentID])`
There are single quotes that are bad.  Switching it to yours is not bad either but I was just curious.